Mathlib/Algebra/MonoidAlgebra/Defs.lean

```diff
@@ -119,6 +119,9 @@ theorem single_apply {a a' : G} {b : k} [Decidable (a = a')] :
 @[simp]
 theorem single_eq_zero {a : G} {b : k} : single a b = 0 ↔ b = 0 := Finsupp.single_eq_zero
 
+@[simp]
+theorem single_ne_zero {a : G} {b : k} : single a b ≠ 0 ↔ b ≠ 0 := single_eq_zero.not
```

Comment:
```suggestion
theorem single_ne_zero {a : G} {b : k} : single a b ≠ 0 ↔ b ≠ 0 := Finsupp.single_ne_zero
```
since this makes it a little clearer that the APIs are in sync